playwright/e2e/suites/accession.disable.ts

```diff
@@ -0,0 +1,158 @@
+
+test.setTimeout(60000);
+test.beforeEach(async ({ context }, testInfo) => {
+  // Make all requests look like they are associated with an existing login session
+  // so we don't have to depend on a Keycloak server to run the test suite. The
+  // session value here is the base64-encoded session ID from dump/session.sql.
+  await context.addCookies([
+    { name: 'SESSION', value: 'Mjc2NzE0YWQtYWIwYS00OGFhLThlZjgtZGI2NWVjMmU5NTBh', url: 'http://127.0.0.1:3000' },
+  ]);
+
+  // Pick the new/fake "now" for the test.
+  const testDate = new Date('January 1 2024 12:00:00Z-08:00').valueOf();
+
+  // Update the Date accordingly in your test pages
+  await context.addInitScript(`{
+    // Extend Date constructor to default to testDate
+    Date = class extends Date {
+      constructor(...args) {
+        if (args.length === 0) {
+          super(${testDate});
+        } else {
+          super(...args);
+        }
+      }
+    }
+    // Override Date.now() to start from testDate
+    const __DateNowOffset = ${testDate} - Date.now();
+    const __DateNow = Date.now;
+    Date.now = () => __DateNow() + __DateNowOffset;
+  }`);
+});
+
+test('Add An Accession', async ({ page }, testInfo) => {
+  await page.goto('http://127.0.0.1:3000');
+
+  await waitFor(page, '#home');
+
+  await page.getByRole('button', { name: 'Seeds' }).click();
+  await page.getByRole('button', { name: 'Accessions' }).click();
+  await page.getByRole('button', { name: 'Add Accession' }).click();
+  await page.getByPlaceholder('Search or Select...').click();
+  await page.locator('li').filter({ hasText: 'Coconut' }).locator('div').click();
+  await page.getByLabel('Collection Date *').click();
+  await page.getByLabel('Choose date', { exact: true }).click();
+  await page.getByRole('gridcell', { name: '1', exact: true }).click();
+  await page.getByPlaceholder('Collectors').click();
+  await page.getByLabel('Close').click();
+  await page.getByPlaceholder('Collectors').click();
+  await page.getByPlaceholder('Collectors').fill('Alex');
+  await page.locator('#collectionSiteName').getByRole('textbox').click();
+  await page.locator('#collectionSiteName').getByRole('textbox').fill("Alex's Mansion");
+  await page.locator('#collectionSiteLandowner').getByRole('textbox').click();
+  await page.locator('#collectionSiteLandowner').getByRole('textbox').fill('Ashtyn');
+  await page.getByRole('button', { name: 'Add GPS Coordinates' }).click();
+  await page.locator('#gpsCoords0').getByRole('textbox').click();
+  await page.locator('#gpsCoords0').getByRole('textbox').fill('2, 4');
+  await page.locator('#addGpsCoordsButton').click();
+  await page.locator('#gpsCoords1').getByRole('textbox').click();
+  await page.locator('#gpsCoords1').getByRole('textbox').fill('8, 8dfdsf');
+  await page.locator('#location').getByRole('img').click();
+  await page.getByText('garage').click();
+  await page.locator('#sub-location path').click();
+  await page.locator('#sub-location path').click();
```

Comment:
was this supposed to click a specific sub-location?

Reply:
This test is a WIP and is disabled.  Will come through and re-enable in a separate PR when it's ready.